lib/runtime/formatter.js

```diff
@@ -0,0 +1,43 @@
+/** @babel */
+
+import { client } from '../connection'
+
+const { format } = client.import(['format'])
+
+
+// @TODO: Once DocumentFormat.jl comes to be able to handle options to specify indents,
+//        we can extract some information from the status-bar tile consumed by atom-indent-detective,
+//        or more safely, make atom-indent-detective provide the service and use it here ?
```

Comment:
We can just query `atom.workspace.getActiveTextEditor().getTabLength()`, no?

Reply:
I thought it would be great if we can use `getIndent` in `atom-indent-detective` on each call of the formatting command, but I think `getTabLength()` API would be simple better.